docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>I think we can drop "to make the API usable".</p><p> </p><p>"If you intend to make custom plots ..." -&gt; "We'll use matplotlib and seaborn for custom plots in this guide, but you can use your preferred tool". (the current text makes it sound like matplotlib and seaborn are required to make custom plots).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.